commands/resource_commands/deploy

```diff
@@ -0,0 +1,247 @@
+        models+=("$yaml_properties")
+        models_name+=("$model_name")
+    done
+    
+    model_parsed_data=""
+    x=0
+    for model in "${models[@]}"; do
+
+        model_with_tab=$(echo "$model" | awk '{print "    " $0}')
+
+        model_parsed_data="$model_parsed_data
+  - name: \"${models_name[$x]}\"
+$model_with_tab"
+
+        x=$x+1
+
+    done
+}
+
+
+take_relation_input() {
+   
+    
+    local models=("$@")
+    RELATIONS=""
+    while true; do
+
+        local relation_type=$(gum choose --header "Please select the relation type" $BELONGS_TO $HAS_MANY $HAS_MANY_THROUGH $HAS_ONE $REFRENCES_MANY) || exit $?
+        status_log $TYPE_DEFAULT  "relation model template: ${relation_type}"
+
+        local source_model=$(gum choose --header "Please select source model" "${models[@]}") || exit $?
+        status_log $TYPE_DEFAULT  "source model: ${source_model}"
+
+        local target_model=$(gum choose --header "Please select target model" "${models[@]}") || exit $?
+        status_log $TYPE_DEFAULT  "destination model: ${target_model}"
+
+        local foreign_key=$(prompt_for_input_with_validation "Foreign key name to define on the source model: " "Please enter foreign key" "$non_empty_regex" "please select valid foreign key") || exit $?
+        status_log $TYPE_DEFAULT  "foreign_key: ${foreign_key}"
+
+        local relation_name=$(prompt_for_input_with_validation "Relation name: " "Please enter relation name" "$non_empty_regex" "please select valid relation name") || exit $?
+        status_log $TYPE_DEFAULT  "relation_name: ${relation_name}"
+
+        local register_inclusion_resolver=$(gum choose --header "Allow Customers queries to include data from related Employees instances?" "Yes" "No") || exit $?
+        status_log $TYPE_DEFAULT  "register_inclusion_resolver: ${register_inclusion_resolver}"
+        
+        if [ "$register_inclusion_resolver" == "Yes" ]; then
+            register_inclusion_resolver=true
+        else
+            register_inclusion_resolver=false
+        fi
+
+        RELATIONS="$RELATIONS
+  - name: \"$relation_name\"
+    spec:
+      relationName: \"$relation_name\"
+      sourceModel: \"$source_model\"
+      destinationModel: \"$target_model\"
+      foreignKeyName: \"$foreign_key\"
+      relationType: \"$relation_type\"
+      registerInclusionResolver: $register_inclusion_resolver
+      "
+
+        status_log $TYPE_INFO "Do you want to add another rule?"
+        local another_rule=$(gum choose "Yes" "No") || exit $?
+        
+        if [ "$another_rule" == "No" ]; then
+            return 0
+        fi
+
+    done
+    
+}
+
+deploy_db_file_template() {
+
+    if [ "$(kubectl config current-context)" = "minikube" ]; then
+        echo "patching for minikube..."
+        MINIKUBE_PATCH=" --set gras.ssl=false --set grapi.ssl=false --set gruim.ssl=false "
+    else
+        MINIKUBE_PATCH=""
+    fi
+
+    echo "check xrds are available"
```

Comment:
all the checks here are not important.
this verifies if grapple is installed...
we should check the installation with "grpl cluster status"
to quickly check the status of the cluster.
but all this verification code here is not required...

Reply:
grpl cluster status command isn't available yet, that is why I added these checks. 
I'll remove then for now and use cluster status command once is implemented